File: app/src/main/java/com/neome/java/api/meta/base/dto/DefnFieldText.java
// neome.ai API - do not change
//

package com.neome.java.api.meta.base.dto;

import com.neome.java.api.meta.base.Types.MetaIdField;

import org.jetbrains.annotations.Nullable;

@SuppressWarnings("unused")
public class DefnFieldText extends DefnFieldEditableText
{
  @Nullable
  public Long maxCharCount;

  @Nullable
  public MetaIdField maxCharCountFieldId;

  @Nullable
  public Long maxCharCountVar;

  @Nullable
  public Long minCharCount;

  @Nullable
  public MetaIdField minCharCountFieldId;

  @Nullable
  public Long minCharCountVar;

  @Nullable
  public DefnDtoTextValidationPattern validationPattern;
}

File: app/src/main/java/com/neome/api/meta/base/dto/DefnFieldText.kt
// neome.ai API - do not change
//
@file:Suppress("unused", "MemberVisibilityCanBePrivate", "CanBeParameter", "PropertyName")


package com.neome.api.meta.base.dto

import com.neome.api.meta.base.dto.DefnDtoTextValidationPattern
import com.neome.api.meta.base.dto.DefnFieldEditableText
import com.neome.api.meta.base.Types.MetaIdField

interface DefnFieldText : DefnFieldEditableText
{
  val maxCharCount: Long?
  val maxCharCountFieldId: MetaIdField?
  val maxCharCountVar: Long?
  val minCharCount: Long?
  val minCharCountFieldId: MetaIdField?
  val minCharCountVar: Long?
  val validationPattern: DefnDtoTextValidationPattern?
}